examples/kafka-streams-examples/README.md

```diff
@@ -0,0 +1,230 @@
+# Confluent's Kafka Streams Examples
```

Comment:
@arrawatia I'm happy to do that.  Where should that live then?

What I'd suggest though is that each example sub-folder should contain a README that points to the docs/tutorials folder and/or docs.confluent.io.  The reason is that, arguably, some people (including me, cough) are browsing this repository on GitHub, and it's wasn't super-obvious to me which folder here (GitHub) maps to which HTML page under the docs.confluent.io.

WDYT?